types/ember-cli-fastboot/test/ember-cli-fastboot-tests.ts

```diff
@@ -0,0 +1,22 @@
+import { assertType } from './assert';
+import { FastBoot as FastBootService, FastbootRequest, Shoebox } from 'ember-cli-fastboot/services/fastboot';
+
+/** type assertions for global FastBoot object */
+assertType<FastBootService | undefined>(FastBoot);
+
+/** type assertions for fastboot service & other classes */
+const instance = new FastBootService();
```

Comment:
I'd stick with syntax that is legal in one way or another:

```suggestion
declare const instance: FastBootService;
```

```suggestion
const instance = FastBootService.create();
```